src/main/resources/templates/scala-lang/scalaClassGraphqlEnum.ftl

```diff
@@ -2,6 +2,8 @@
 package ${package}
 
 </#if>
+import com.fasterxml.jackson.core.`type`.TypeReference
```

Comment:
@jxnu-liguobin, do you need to wrap this import into `<#if serializationLibrary == 'JACKSON'>`?

Reply:
yes, will change